guide/guide/src/main/scala/io/udash/web/guide/views/ext/demo/bootstrap/NavbarDemo.scala

```diff
@@ -0,0 +1,73 @@
+package io.udash.web.guide.views.ext.demo.bootstrap
+
+import io.udash._
+import io.udash.bootstrap.nav.{UdashNav, UdashNavbar}
+import io.udash.bootstrap.utils.BootstrapStyles
+import io.udash.css.CssView
+import io.udash.logging.CrossLogging
+import io.udash.properties.seq.SeqProperty
+import io.udash.web.guide.demos.AutoDemo
+import io.udash.web.guide.styles.partials.GuideStyles
+import org.scalajs.dom.Event
+import scalatags.JsDom
+
+object NavbarDemo extends AutoDemo with CrossLogging with CssView {
+
+  import JsDom.all._
+
+  trait NavbarPanel {
+    def title: String
+
+    def content: String
+  }
+
+  object NavbarPanel extends HasModelPropertyCreator[NavbarPanel]
+
+  final case class DefaultNavbarPanel(override val title: String, override val content: String) extends NavbarPanel
+
+  private val (rendered, source) = {
+    /*
```

Comment:
They look completely different in the UI and serve different purposes. I'll leave them separate since they are separate components event on the official bootstrap4 docs page.